<launch>

  <!-- GDB functionality -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <!-- Verbose mode -->
  <arg name="verbose" default="false" />
  <arg unless="$(arg verbose)" name="command_args" value="" />
  <arg     if="$(arg verbose)" name="command_args" value="true" />
  
  <arg name="robot_name" default="romeo" />
  <arg name="test_step" default="0.02" />

  <!-- Launch pick and place -->
  <node name="pick_place_moveit_romeo" launch-prefix="$(arg launch_prefix)" pkg="pick_place_moveit"
  type="pick_place_moveit" output="screen" args="$(arg command_args) --robot_name=$(arg robot_name) test_step=$(arg test_step)">
    <rosparam command="load" file="$(find romeo_moveit_config)/config/kinematics.yaml"/>
    <rosparam command="load" file="$(find moveit_simple_grasps)/config/romeo_grasp_data.yaml"/>
  </node>

</launch>
